setup.py

```diff
@@ -570,14 +580,16 @@ def pxd(name):
     '_libs.tslibs.timezones': {
         'pyxfile': '_libs/tslibs/timezones'},
     '_libs.testing': {
-        'pyxfile': '_libs/testing'},
+        'pyxfile': '_libs/testing',
+        'include': []},
```

Comment:
what is the difference betwee not passing include and an empty list? (from a compilation perspective)?

Reply:
I think this can be removed IINM.

Reply:
Without the 'include' key, it gets set to a default of `common_include`.  This has two unwanted side effects.  First: the search path for numpy now includes src/numpy.pxd which we should move away from.  Second: making it explicit avoids a problem a year from now when we see that `common_include` is added to `_libs.testing` but no one remembers that it isn't actually needed.